src/components/Staking/StakingConsiderations.tsx

```diff
@@ -47,7 +48,7 @@ const ListItem = styled.li`
   box-sizing: border-box;
   position: relative;
   height: 2rem;
-  ${({ theme, active }) =>
+  ${({ theme, active }: { theme: DefaultTheme; active: boolean }) =>
```

Comment:
In this cases where we have custom props, lets pass this custom prop types like this:
```ts
const ListItem = styled.li<{ active: boolean }>`
```
that way you don't need to define the types as you did in this line.